packages/common/src/store/buy-usdc/slice.ts

```diff
@@ -45,13 +45,11 @@ const slice = createSlice({
       action: PayloadAction<{
         purchaseInfo: PurchaseInfo
         provider: USDCOnRampProvider
-        onSuccess?: OnSuccess
```

Comment:
I could have sworn we were using this as a callback mechanism. I wonder when that changed....
I did read through the current code to verify that other sagas just watch the onFlowSucceeded action from this slice. So I suppose we're okay 🤷

Reply:
Yeah I too was suspect - I did attempt to check all references of this action, might be worth you sanity checking me?